hnn_core/gui/gui.py

```diff
@@ -1726,10 +1726,11 @@ def run_button_clicked(widget_simulation_name, log_out, drive_widgets,
             simulation_status_bar.value = simulation_status_contents[
                 'finished']
 
-            sim_names = [sim_name for sim_name
-                         in simulation_data]
-            simulations_list_widget.options = sim_names
-            simulations_list_widget.value = sim_names[0]
+            run_simulations = [sim_name for sim_name in simulation_data
+                               if simulation_data[sim_name]['net'] is not None]
```

Comment:
I think I prefer `sim_names` here still, but otherwise this looks great!

Reply:
done